### ISSUE TITLE
Missing file or incorrect path: 'class_labels_indices.csv'

### ISSUE THREAD

**Issue by dotgrid** [bytedance/uss#4]
When I run the example code:
```bash
uss -i "harry_potter.flac"
```

I receive an error:
```bash
...

FileNotFoundError: [Errno 2] No such file or directory: '/Users/paul/panns_data/class_labels_indices.csv'
```

**Comment by SoftologyPro** [comment]
Same error for me (running under Windows)
```
  File "D:\Python\lib\site-packages\panns_inference\config.py", line 16, in <module>
    with open(labels_csv_path, 'r') as f:
FileNotFoundError: [Errno 2] No such file or directory: 'C:\\Users\\UserName/panns_data/class_labels_indices.csv'
```

**Comment by Gitterman69** [comment]
same here!

**Comment by qiuqiangkong** [comment]
Hi all, for Windows users, one solution is to manually download the file [class_labels_indices.csv](http://storage.googleapis.com/us_audioset/youtube_corpus/v1/csv/class_labels_indices.csv) and put it in "~/panns_data/class_labels_indices.csv". For Mac & Linux users this file will be downloaded automatically.

**Comment by SoftologyPro** [comment]
Downloaded the file.
Next error is about the same file missing from
`C:\\Users\\UserName\\.cache\\uss\\metadata\\class_labels_indices.csv`
so I downloaded it in there too.
Then the next error...
`FileNotFoundError: [Errno 2] No such file or directory: 'C:\\Users\\UserName\\.cache\\uss\\scripts\\ss_model=resunet30,querynet=at_soft,data=full.yaml'`

**Comment by qiuqiangkong** [comment]
Are you using Windows?

**Comment by qiuqiangkong** [comment]
@SoftologyPro Are you using Windows?

**Comment by SoftologyPro** [comment]
Yes

**Comment by sandip183** [comment]
Downloading 
 https://zenodo.org/record/3987831/files/Cnn14_mAP%3D0.431.pth?download=1
https://zenodo.org/record/3987831/files/Cnn14_DecisionLevelMax_mAP%3D0.385.pth?download=1

and putting it under 
.cache/panns/   resolved this for me

**Comment by SoftologyPro** [comment]
> Downloading https://zenodo.org/record/3987831/files/Cnn14_mAP%3D0.431.pth?download=1 https://zenodo.org/record/3987831/files/Cnn14_DecisionLevelMax_mAP%3D0.385.pth?download=1
> 
> and putting it under .cache/panns/ resolved this for me

Did not seem to make a difference here.  Same error for me.

`FileNotFoundError: [Errno 2] No such file or directory: 'C:\\Users\\UserName\\.cache\\uss\\scripts\\ss_model=resunet30,querynet=at_soft,data=full.yaml'`

**Comment by sandip183** [comment]
I think copying the file from here https://github.com/bytedance/uss/tree/master/scripts/train to the .cache\\uss\\scripts\\ 
shall resolve it..I copied the missing file as and when it threw missing error.

**Comment by SoftologyPro** [comment]
> I think copying the file from here https://github.com/bytedance/uss/tree/master/scripts/train to the .cache\uss\scripts\ shall resolve it..I copied the missing file as and when it threw missing error.

Yes, that fixed it.  You do not have to manually download the model files.  It handles that for you (you will need to put wget.exe in your path for the downloads to work).

Download the missing yaml file it complains about 
`https://github.com/bytedance/uss/blob/master/scripts/train/ss_model%3Dresunet30%2Cquerynet%3Dat_soft%2Cdata%3Dfull.yaml`
and save it into the path it shows.

Then everything should run fine.
